frontend/src/components/screencast.ts

```diff
@@ -76,6 +76,9 @@ export class Screencast extends LitElement {
   @property({ type: String })
   crawlId?: string;
 
+  @property({ type: Array })
+  dataList: Array<ScreencastMessage> = [];
```

Comment:
Suggestion to use private `state`, since `dataList` is internal
```suggestion
  @state()
  private dataList: Array<ScreencastMessage> = [];
```